components/app_update/esp_ota_ops.c

```diff
@@ -197,13 +197,20 @@ esp_err_t esp_ota_write(esp_ota_handle_t handle, const void *data, size_t size)
         return ESP_ERR_INVALID_ARG;
     }
 
+    if(size == 0)
+    {
+        ESP_LOGD(TAG, "write data size is 0");
+        return ESP_OK;
+    }
```

Comment:
```suggestion
    if (size == 0) {
        ESP_LOGD(TAG, "write data size is 0");
        return ESP_OK;
    }
```

Reply:
@kohait00 You missed my suggestion in the previous update. Please see our coding style guidelines [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/contribute/style-guide.html).